errors/errors_test.go

```diff
@@ -67,12 +67,12 @@ func (s *errorsTestSuite) TestErrorIs() {
 		},
 		"successful comparison to a wrapped error": {
 			a:      ErrUnauthorized,
-			b:      errorsmod.Wrap(ErrUnauthorized, "gone"),
```

Comment:
This was broken, as there were no import for `errorsmod`.
Checking if CI checks the errors package properly.